.github/ISSUE_TEMPLATE/feature---enhancement-proposal.md

```diff
@@ -12,14 +12,14 @@ Please fill in *all* the questions below and don't remove any of them.
 Proposals not following the template below will be closed immediately.
 -->
 
-**Describe the project you are working on:**
+### Describe the project you are working on:
```

Comment:
Since you're making these headers, please remove the trailing colon.

That would also solve the ugly "?:" situation on the two last ones.

Reply:
It's a ternary operator! :stuck_out_tongue: